ldon/rustfmt.toml

```diff
@@ -0,0 +1,98 @@
+# title: Yatima Standard Rustfmt Configuration
+# author: John C. Burnham <john@yatima.io>
+
+# For additional information about Rustfmt options see:
+# https://rust-lang.github.io/rustfmt/?version=v1.5.1&
+
+# General
+version              = "One"              # rustfmt 2.0 is not released yet
+edition              = "2021"             # Rust edition
+unstable_features    = false              # don't use unstable rustfmt features
+newline_style        = "Unix"             # Never allow Windows' \n\r newlines
```

Comment:
Why isn't `newline_style` auto? It sounds suspicious to have differently formatted/sized source files on different platforms, but since auto is the default I'm wondering if there's a specific reason.

Reply:
Windows-style newlines are bad and we should just forbid them. It's easy for someone developing on Windows to set their editor to mimic Unix and we should encourage them to do that